lib/client.js

```diff
@@ -304,6 +304,14 @@ Client.prototype.disconnect = function(cb) {
 
 /**
  * Get client metadata.
+ * <p/>
+ * <b>Note:</b> using a <code>metadataOptions.topic</code> parameter has a potential side-effect.
+ * A Topic object will be created, if it did not exists yet, with default options
```

Comment:
I have mixed feelings about using HTML markup in this comment when it isn't used in other places. At the very least, can we remove the `<p/>` and just leave the comment formatted as the other ones are?